integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/validation/OneOfValidator.java

```diff
@@ -0,0 +1,23 @@
+package no.difi.meldingsutveksling.validation;
+
+import lombok.extern.slf4j.Slf4j;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+@Slf4j
+public class OneOfValidator implements ConstraintValidator<OneOf, String> {
+
+    private Set<String> acceptedValues;
+
+    public void initialize(OneOf constraint) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1161)